### PR TITLE
WWA Keyboards

### PR DESCRIPTION
**Pull request by spacehangover**
Added support to WWA Macropads, Argentinian Keyboards

## Description

Added suppor to Helios and Kepler models
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ x] I have added tests to cover my changes.
- [ x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/wwa/helios/config.h
Comment (waffle87) [inline keyboards/wwa/helios/config.h:1]: This file should be removed.

File: keyboards/wwa/helios/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# helios
+
+![helios](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [spacehangover](https://github.com/spacehangover)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
Comment (waffle87) [inline keyboards/wwa/helios/readme.md:9]: Please fill these fields out.

File: keyboards/wwa/kepler/config.h
Comment (waffle87) [inline keyboards/wwa/kepler/config.h:1]: This file should be removed.

File: keyboards/wwa/helios/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+     * ┌───┬───┬───┬───┐
+     * │ 7 │ 8 │ 9 │ / │
+     * ├───┼───┼───┼───┤
+     * │ 4 │ 5 │ 6 │ * │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │ - │
+     * ├───┼───┼───┼───┤
+     * │ 0 │ . │Ent│ + │
+     * └───┴───┴───┴───┘
+     */
```
Comment (waffle87) [inline keyboards/wwa/helios/keymaps/default/keymap.c:17]:
```suggestion
```

File: keyboards/wwa/helios/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+     * ┌───┬───┬───┬───┐
+     * │ 7 │ 8 │ 9 │ / │
+     * ├───┼───┼───┼───┤
+     * │ 4 │ 5 │ 6 │ * │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │ - │
+     * ├───┼───┼───┼───┤
+     * │ 0 │ . │Ent│ + │
+     * └───┴───┴───┴───┘
+     */
```
Comment (waffle87) [inline keyboards/wwa/helios/keymaps/via/keymap.c:17]:
```suggestion
```

File: keyboards/wwa/kepler/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+     * ┌───┬───┬───┬───┐
+     * │ 7 │ 8 │ 9 │ / │
+     * ├───┼───┼───┼───┤
+     * │ 4 │ 5 │ 6 │ * │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │ - │
+     * ├───┼───┼───┼───┤
+     * │ 0 │ . │Ent│ + │
+     * └───┴───┴───┴───┘
+     */
```
Comment (waffle87) [inline keyboards/wwa/kepler/keymaps/default/keymap.c:17]:
```suggestion
```

File: keyboards/wwa/kepler/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+     * ┌───┬───┬───┬───┐
+     * │ 7 │ 8 │ 9 │ / │
+     * ├───┼───┼───┼───┤
+     * │ 4 │ 5 │ 6 │ * │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │ - │
+     * ├───┼───┼───┼───┤
+     * │ 0 │ . │Ent│ + │
+     * └───┴───┴───┴───┘
+     */
```
Comment (waffle87) [inline keyboards/wwa/kepler/keymaps/via/keymap.c:17]:
```suggestion
```

File: keyboards/wwa/kepler/readme.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# kepler
+
+![kepler](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [spacehangover](https://github.com/spacehangover)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
Comment (waffle87) [inline keyboards/wwa/kepler/readme.md:9]: Please fill these fields out.

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/wwa/helios/readme.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# helios
+
+![helios](https://imgur.com/a/mXWRgCt)
+
+First model of the Argentinian based keyboard manufacturer WWA
+
+* Keyboard Maintainer: [spacehangover](https://github.com/spacehangover)
+* Hardware Supported: RP2040 Handwired
+* Hardware Availability: https://www.amazon.com/dp/B08ZSKMJJD/ref=sspa_dk_detail_6?psc=1&pd_rd_i=B08ZSKMJJD&pd_rd_w=K8K1X&content-id=amzn1.sym.08ba9b95-1385-44b0-b652-c46acdff309c&pf_rd_p=08ba9b95-1385-44b0-b652-c46acdff309c&pf_rd_r=3W158XFKMFJTGARYNNT6&pd_rd_wg=ADHNt&pd_rd_r=7e2afce9-17d9-46a7-b196-5be2ab375f40&s=pc&sp_csd=d2lkZ2V0TmFtZT1zcF9kZXRhaWxfdGhlbWF0aWM&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUEyQTU4SElNN0VVNVdXJmVuY3J5cHRlZElkPUEwNTA3MTMwMlZDSDVMVEQwQ0ZPTSZlbmNyeXB0ZWRBZElkPUEwMDgzMTI4MUE4UkxCWEpCTVk5ViZ3aWRnZXROYW1lPXNwX2RldGFpbF90aGVtYXRpYyZhY3Rpb249Y2xpY2tSZWRpcmVjdCZkb05vdExvZ0NsaWNrPXRydWU=
```
Comment (waffle87) [inline keyboards/wwa/helios/readme.md:9]:
```suggestion
* Hardware Availability: RP2040 Pico
```
Or any sort of link to project specific info, like a case or something.

**Review by waffle87 — CHANGES_REQUESTED** [review]
As these are both handwired keyboards, the `wwa` directory should be moved under `keyboards/handwired`.

File: keyboards/handwired/wwa/kepler/readme.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+# kepler
+
+![kepler](https://i.imgur.com/UD1J92Eh.jpeg)
+
+*A short description of the keyboard/project*
```
Comment (fauxpark) [inline keyboards/handwired/wwa/kepler/readme.md:5]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by waffle87 — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**